src/test/tables/joinoperators/anti_int4.tbl

```diff
@@ -0,0 +1,3 @@
+a
+int
+123456
```

Comment:
Test with input data that generates more than one output row

Reply:
done :)

Reply:
happy with everything now, @mrzzzrm ? (: